### ISSUE TITLE
Sorry. not sure how to contact. Do you have email?

### ISSUE THREAD

**Issue by cocoonkid** [x-Ai/CSIRTool#1]
Sorry. not sure how to contact. Do you have email?

**Comment by x-Ai** [comment]
Hi brother, are you having any problems?

**Comment by cocoonkid** [comment]
Yes, but better via e-mail or similar. Not for the public. Thanks in advance.

**Comment by cocoonkid** [comment]
Please. Don't worry it's nothing weird haha. Just not for public consumption.
